apps/docs/src/getting-started.md

```diff
@@ -17,16 +17,18 @@ To begin, you need to add the `fuels` dependency to your project. You can do thi
 
 ::: code-group
 
-```sh [npm]
-npm install fuels --save
+```sh-vue [npm]
+npm install fuels@{{fuelsInstallVersion}} --save
 ```
 
-```sh [pnpm]
-pnpm add fuels
+```sh-vue [pnpm]
```

Comment:
What is ```sh-vue ?

Reply:
Is there any chance this can be misinterpreted on the docs hub? 🤔

Reply:
I know I said it first, but I don't know if there's any advantage in using a new variable then.

Perhaps we could use the version from `@fuel-ts/versions` and call it a day.

What do you think?

Reply:
Let's not discard the option of using the same variable, as it could solve the issue without relying on external things.

In this approach of using different variables, do we need to wait for https://github.com/FuelLabs/docs-hub/pull/247?

If that doesn't get merged in time, what would we do?

Reply:
@petertonysmith94 As we spoke, let's go without the second variable. 🙁